macos/FluentUITestViewControllers/TestColorViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+import FluentUI
+
+class TestColorViewController: NSViewController {
+	override func loadView() {
+		let containerView = NSView()
+		let scrollView = NSScrollView()
+		scrollView.translatesAutoresizingMaskIntoConstraints = false
+		scrollView.hasVerticalScroller = true
+		containerView.addSubview(scrollView)
+		
+		let colorsStackView = NSStackView()
+		colorsStackView.translatesAutoresizingMaskIntoConstraints = false
+		colorsStackView.orientation = .vertical
+		colorsStackView.alignment = .leading
+		
+		for color in Colors.Palette.allCases {
+			let colorView = ColorRectView(color: color.color)
+			colorView.invalidateIntrinsicContentSize()
+			let textView = NSTextField(labelWithString: color.name)
+			textView.font = .systemFont(ofSize: 18)
+			let rowStackView = NSStackView()
+			rowStackView.orientation = .horizontal
+			rowStackView.spacing = 20.0
+			rowStackView.addArrangedSubview(colorView)
+			rowStackView.addArrangedSubview(textView)
+			colorsStackView.addArrangedSubview(rowStackView)
+		}
+		
+		let documentView = NSView()
+		documentView.translatesAutoresizingMaskIntoConstraints = false
+		scrollView.documentView = documentView
+		documentView.addSubview(colorsStackView)
+		
+		NSLayoutConstraint.activate([
+			containerView.heightAnchor.constraint(equalTo:scrollView.heightAnchor),
+			containerView.widthAnchor.constraint(equalTo:scrollView.widthAnchor),
+
+			colorsStackView.topAnchor.constraint(equalTo: documentView.topAnchor, constant: 10.0),
```

Comment:
minor: split the 10 out into a constant - avoid duplication and assign meaning to the value by giving it a name